lib/elements/element.py

```diff
@@ -137,7 +139,12 @@ def set_param(self, name, value):
     def get_style(self, style_name, default=None):
         style = simplestyle.parseStyle(self.node.get("style"))
         if (style_name not in style):
-            return default
+            # possibly the specific style is set as a seperate attribute
```

Comment:
Awesome, thanks for this!  I keep forgetting that styles previously were individual attributes.

Reply:
Thanks to the youtube video of Azn Mike! I wouldn't have thought of it otherwise.